core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -295,12 +308,18 @@ struct ComplexReducerSizeCalculator {
   }
 };
 
+template <typename Policy>
+auto default_tuned_version_of(const Policy& policy) {
+  return policy;
```

Comment:
I am not clear if its a good idea that the default thing still creates a new policy. Doesn't this do that since it not returns a auto&?